packages/discovery-react-components/src/components/CIDocument/components/CIDocument/CIDocument.tsx

```diff
@@ -611,13 +632,21 @@ function getActiveIndex(activeIds: string[], highlightedList: any[]): number {
 
 function getActiveElement(activeIds: string[], itemList: any[]): any {
```

Comment:
the return type of this function is `boolean` right? not `any`

Reply:
No, it gives details about the active element, not boolean.